src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy

```diff
@@ -759,13 +759,74 @@ class LinkSpec extends HealthCheckSpecification {
             antiflap.portUp(isl.srcSwitch.dpId, isl.srcPort)
             Wrappers.wait(discoveryInterval + WAIT_OFFSET) {
                 def links = northbound.getAllLinks()
-                assert islUtils.getIslInfo(links, isl.reversed).get().state == IslChangeType.DISCOVERED
-                assert islUtils.getIslInfo(links, isl).get().state == IslChangeType.DISCOVERED
+                assert islUtils.getIslInfo(links, isl.reversed).get().state == DISCOVERED
+                assert islUtils.getIslInfo(links, isl).get().state == DISCOVERED
             }
         }
         database.resetCosts()
     }
 
+    @Tidy
+    def "The src/dst switches are valid when isl is discovered in one direction only"() {
```

Comment:
I don't like this naming since it is very narrow. Test should describe a certain situation that we are testing, more broad scenario. Right now this looks like a pin-point test that is created to test a specific defect, very low value.
Describe a real-life situation/use-case: `System detects a 1-way ISL as a Failed ISL` -> then as part of this broad test add required verifications

Reply:
Make sense.Renamed.